google-cloud-examples/src/main/java/com/google/cloud/examples/spanner/snippets/DatabaseAdminClientSnippets.java

```diff
@@ -70,7 +71,12 @@ public Database createDatabase(String instanceId, String databaseId) {
                     + "  AlbumTitle   STRING(MAX)\n"
                     + ") PRIMARY KEY (SingerId, AlbumId),\n"
                     + "  INTERLEAVE IN PARENT Singers ON DELETE CASCADE"));
-    Database db = op.waitFor().getResult();
+    Database db;
+    try {
+      db = op.get();
+    } catch (Exception e) {
+      throw SpannerExceptionFactory.newSpannerException(e);
```

Comment:
Why would user create a SpannerException? Instead is it possible to have the ExecutionException returned by this method wrap a SpannerExceptiojn?

Reply:
@vkedia We realize that there is more work to do in gax before we can get this done. Please hold off reviewing for a bit. I'll change the title to WIP as well. Sorry that I should have let you know earlier.